### ISSUE TITLE
the @onInit-function don't fire

### ISSUE THREAD

**Issue by panjon** [tinymce/tinymce-vue#8]
Hi
Since version 1.0.1 the @onInit-function don't fire anymore. @onFocus-function fire as expected.
```
<template>
   <tinymceeditor @onInit="init" @focus="focus" ...></tinymceditor>
</template>
<script>
.......
export default {
.......
methods: {
            focus: function (e) {
                console.log("editor focus")
            },
            init: function (e) {
                console.log("editor init")
            }
        }
    }
</script>

```
Is this an known issue?
Regards
Jonas

**Comment by fyrkant** [comment]
Yeah you are right, this doesn't work anymore since I changed to bind all handlers after the editor has initialized, so the init will already have fired when the onInit handler gets bound. I'm not totally sure why I did that change. 🤔 😅  

I'll look into it! While I'm doing that you should be able to bind the init event like this: 

```js
<editor :init="{ setup: function (editor) { editor.on('init', handlerFunc); } }"
```

**Comment by fyrkant** [comment]
Fixed with 1.0.4 release!

**Comment by mkwsra** [comment]
@fyrkant I faced the same problem, none of the events were fired, and I'm using the version, `2.0.0`
